dashboard/test/models/user_test.rb

```diff
@@ -1365,4 +1365,36 @@ def create_user_with_username(username)
     create(:plc_user_course_enrollment, plc_course: (create :plc_course), user: user)
     assert user.should_see_inline_answer?((create :script_level))
   end
+
+  test 'do not show race interstitial to teacher' do
+    teacher = create :teacher
+    refute teacher.show_race_interstitial?
+  end
+
+  test 'do not show race interstitial to users under 13' do
+    student = User.create(@good_data_young)
+    refute student.show_race_interstitial?
+  end
+
+  test 'do not show race interstitial to users less than one week old' do
+    student = create :student
+    student.created_at = DateTime.now.to_date - 3
+    refute student.show_race_interstitial?
+  end
+
+  test 'do not show race interstitial to users that have already entered race information' do
+    # also covers the case where the dialog has been dismissed
+    student = create :student
+    student.races = %w(white black)
+    refute student.show_race_interstitial?
+
+    student.races = %w(closed_dialog)
```

Comment:
nit: This is two separate tests. And the comment in `users.rb` does not suggest that `closed_dialog` is a valid value.